csm/core/controllers/stats.py

```diff
@@ -144,7 +144,7 @@ async def post(self):
             schema = MatricsSchemaValidator()
             user_body = schema.load(await self.request.json(), unknown='EXCLUDE')
         except json.decoder.JSONDecodeError:
-            raise InvalidRequest(message_args="Request body missing")
+            raise InvalidRequest("Request body missing")
         except ValidationError as val_err:
             raise InvalidRequest(f"Invalid request body: {val_err}")
```

Comment:
you can use ValidationErrorFormatter while raising InvalidRequest in case of marshmallow ValidationError.
Please check once.

Reply:
Not required as per discussion